extensions/amp-story/1.0/amp-story.js

```diff
@@ -1064,10 +1064,57 @@ export class AmpStory extends AMP.BaseElement {
       });
     }
 
+    this.buildScreenReaderBackButton_();
+
     // Will resolve when all pages are built.
     return storyLayoutPromise;
   }
 
+  /**
+   * Creates a focusable button for screen readers to navigate back.
+   * @private
+   */
+  buildScreenReaderBackButton_() {
+    const backButton = document.createElement('button');
+    this.element.appendChild(backButton);
+    backButton.classList.add('i-amphtml-story-screen-reader-back-button');
+    backButton.setAttribute('aria-label', 'Previous page');
+
+    backButton.addEventListener('click', () => {
+      this.previous_();
+    });
+
+    this.storeService_.dispatch(
+      Action.SET_ADVANCEMENT_MODE,
+      AdvancementMode.MANUAL_ADVANCE
+    );
+
+    hideOnFirstPage(
+      Number(this.storeService_.get(StateProperty.CURRENT_PAGE_INDEX))
+    );
+
+    this.storeService_.subscribe(
+      StateProperty.CURRENT_PAGE_INDEX,
+      (pageIndex) => hideOnFirstPage(pageIndex)
+    );
```

Comment:
You can pass a third parameter `true /** callToInitialize */` which allows you to remove the first manual call to `hideOnFirstPage` that you have right above the storeservice subscribe. It'll trigger the callback with whatever `CURRENT_PAGE_INDEX` is currently set to.